simvue/client.py

```diff
@@ -165,6 +193,12 @@ def init(self, name=None, metadata={}, tags=[], description=None, folder='/'):
         if response.status_code != 200:
             raise RuntimeError('Unable to create run due to: %s', response.text)
 
+        # Start heartbeat thread
+        self._event = Event()
+        self._thread = Thread(target=self._heartbeat, args=(), daemon=True)
```

Comment:
Should we have a counter to check that the number of threads started running and closed stack up i.e. started = closed + running?

Reply:
What if the user's code creates threads as well?